utils/das_data_sorter.py

```diff
@@ -2,15 +2,31 @@
 import argparse 
 from math import ceil
 
+# averages data within the seconds, according to seconds array 
+def avg_Data(array):
```

Comment:
Curious as to why you moved the function to the top? Generally, this is a good thing. But since you are using a global variable `new_time`, I'm not sure how Python will react if you were to run the `avg_Data` function before `new_time` is defined. 

This problem would be solved if we didn't have such global variables, but we'll slowly teach you some new tricks ;)

Reply:
I just ran it, and it worked just as fine. But I understand what you meant by that, it would be a problem in some cases. I put the function on top because I remembered it being good practice during my Javascript days.